sdk/storage/AzureStorageBlob/Source/Transport/URLSessionTransferManager.swift

```diff
@@ -496,6 +496,7 @@ internal final class URLSessionTransferManager: NSObject, TransferManager, URLSe
                     properties: blobProperties,
                     options: uploadOptions
                 )
+                transfer.uploader?.progress = Int(transfer.bytesTransferred)
```

Comment:
Total bytes to transfer for upload is determined by the file size, so we can't set it (and don't need to).